content/en/docs/user-guides/how-to/plugin-management.md

```diff
@@ -11,22 +11,74 @@ Plugins for `notation` provide functionality and integration with key stores and
+To install a plugin, run the `notation plugin install` command to directly install a plugin either from a URL or from the host file system. This will create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/` if the directory does not exist. The supported plugin file formats are `.zip, .tar.gz` and `single plugin executable file`. Alternatively, if you are using an application that bundles a plugin and the `notation` binary together, such as [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html), see the installation instructions from the vendor.
 
 {{% alert title="Warning" color="warning" %}}
 Before creating the `{plugin-name}` directory, confirm you are using a name that follows the [naming structure](#plugin-naming-structure) for plugins. Plugins that do not follow the naming structure will not be recognized by `notation`.
 {{% /alert %}}
 
-The following example downloads and installs version 0.6.0 of [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin for on macOS with Apple Silicon using the default location for the Notation installation.
+## Usage
 
-{{% alert title="Warning" color="warning" %}}
-The following example only works for version 0.6.0 of the *notation-azure-kv* plugin on macOS with Apple Silicon using the default location for the Notation installation. You will need to update the filenames, location, and commands for other plugins, versions, and platforms. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/user-guides/how-to/directory-structure.md" >}}).
+### Install a plugin from file system:
+
+`notation plugin install --file <file_path>`
+
+### Install a plugin from URL: 
+
+`notation plugin install --sha256sum <digest> --url <HTTPS_URL>`
+
+{{% alert title="plugin" color="info" %}}
+The following examples show how to install each plugin on a Linux AMD64 machine. To install a plugin on other operating systems and architectures, please get the URL or plugin binary file from plugin vendors' website.
 {{% /alert %}}
 
+## Install Notation AWS Signer plugin (v1.0.298)
+
+### Install from file system
+
+```console
+wget https://d2hvyiie56hcat.cloudfront.net/linux/amd64/plugin/latest/notation-aws-signer-plugin.zip
+
+notation plugin install --file notation-aws-signer-plugin.zip
+Successfully installed plugin com.amazonaws.signer.notation.plugin, version 1.0.298
+```
+Upon successful execution, the plugin is copied to Notation's plugin directory.
+
+### Install from URL:
+*(Installation from URL is unsupported.)*
```

Comment:
I assume this is because hashes are not listed on the website. I would suggest omitting this section entirely since it might become obsolete if AWS begin publishing hashes on the website.